source/features/default-branch-button.tsx

```diff
@@ -6,9 +6,14 @@ import ChevronLeftIcon from 'octicon/chevron-left.svg';
 import features from '../libs/features';
 import {groupButtons} from '../libs/group-buttons';
 import getDefaultBranch from '../libs/get-default-branch';
-import {getRepoURL, getCurrentBranch, replaceBranch} from '../libs/utils';
+import {getRepoURL, getCurrentBranch, replaceBranch, parseRoute} from '../libs/utils';
 
 async function init(): Promise<false | void> {
+	// The branch selector will be on `isRepoCommitList()` **unless** you're in a folder/file
+	if (pageDetect.isRepoCommitList() && parseRoute(location.pathname)[6]) {
+		return false;
+	}
```

Comment:
You can move this to `exclude` i think

Reply:
I put it here since I felt it was not so _simple_, will move it.